docs/rule/no-arguments-for-html-elements.md

```diff
@@ -52,12 +52,6 @@ This rule **forbids** the following:
 <img src="@img" >
 ```
 
-```hbs
```

Comment:
There's a line `trying to use an HTML element with block params` above that you'll want to remove too.

Reply:
good point. fixed :)